docs/index.rst

```diff
@@ -376,157 +304,77 @@ In this demo, KSQL is authenticated and authorized to connect to the secured Kaf
-
-2. Let this consumer group run for 2 minutes until Control Center stream
-   monitoring shows the consumer group ``app`` with steady consumption.
-
-3. Add a consumption quota for one of the consumers in the consumer
+8. Add a second consumer ``consumer_app_2`` to the existing consumer
    group ``app``.
 
    .. sourcecode:: bash
 
-          ./scripts/app/throttle_consumer.sh 1 add
-
-   .. note:: You are running a Docker demo environment with all services running on one host, which you would never do in production.  Depending on your system resource availability, sometimes applying the quota may stall the consumer (`KAFKA-5871 <https://issues.apache.org/jira/browse/KAFKA-5871>`__), thus you may need to adjust the quota rate. See the ``./scripts/app/throttle_consumer.sh`` script for syntax on modifying the quota rate.
-
-      -  If consumer group ``app`` does not increase latency, decrease the quota rate
-      -  If consumer group ``app`` seems to stall, increase the quota rate
-
-
-4. View the details of the consumer group ``app`` again,
-   ``consumer_app_1`` now shows high latency, and ``consumer_app_2``
-   shows normal latency.
-
-   .. figure:: images/slow_consumer.png
-      :alt: image
-
-5. In the System Health dashboard, you see that the fetch request
-   latency has likewise increased. This is the because the broker that
-   has the partition that ``consumer_app_1`` is consuming from is taking
-   longer to service requests.
-
-   .. figure:: images/slow_consumer_fetch_latency.png
-      :alt: image
-
-6. Click on the fetch request latency line graph to see a breakdown of
-   produce and fetch latencies through the entire `request
-   lifecycle <https://docs.confluent.io/current/control-center/docs/systemhealth.html>`__.
-   The middle number does not necessarily equal the sum of the
-   percentiles of individual segments because it is the total percentile
-   latency.
-
-   .. figure:: images/slow_consumer_fetch_latency_breakdown.png
-      :alt: image
+          ./scripts/app/start_consumer_app.sh 2
 
-7. **MONITORING -> Consumer lag**: consumer lag is the topic's high water mark (latest offset for the topic that has been written) minus the current consumer offset (latest offset read for that topic by that consumer group). Keep in mind topic write rate and consumer group read rate when considering what the significance of how big is the consumer lag. In the demo, view the consumer lag for the ``app`` consumer group: expect consumer 1 to be have much more lag than consumer 2 because of the throttle you added in an earlier step. 
+9. Let this consumer group run for 2 minutes until |c3|
+   shows the consumer group ``app`` with steady consumption.
+   Notice that the consumers ``consumer_app_1`` and ``consumer_app_2``
+   now share consumption of the partitions in the topic
+   ``wikipedia.parsed``. 
 
-   .. figure:: images/consumer_lag_app.png
+   .. figure:: images/consumer_start_two.png
```

Comment:
Very clear demo on Consumer Group Lag @ybyzek !  Great features in C3 as well!

Reply:
Cool!